app/main.ipynb

```diff
@@ -126,19 +143,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
+    "import numpy as np\n",
+    "\n",
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "\n",
+    "subsequence = array[1:6]\n",
+    "result = subsequence.sum()\n",
```

Comment:
The subsequence sum does not match the expected result of 27. Check the indices used for slicing the array to ensure they match the task requirements.